.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,71 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/sharing-workflows-with-your-organization

Looks like there may be some ways we can share this workflow within the org in an easier way.

Reply:
I'm assuming we do not have a `.github` repo currently? If we do, I don't have permission to see it.

Reply:
Also, following the above documentation, we still have to commit the workflow file in the desired repo after selecting `Set up this Workflow` in the UI. It's a shortcut to get here but this PR step will still need to happen since we can't commit directly to master.

Reply:
> I'm assuming we do not have a `.github` repo currently? If we do, I don't have permission to see it.

We don't. @hemp and I were talking about adding one maybe a week ago? Might as well do it!

Reply:
Also- link to results https://github.com/lifeomic/phc-sdk-py/security/code-scanning?query=tool%3ACodeQL+ref%3Arefs%2Fpull%2F85%2Fhead+is%3Aopen+

Reply:
@joedimarzio and @bishopb: Are you guys going to do the `.github` approach or are we just going to go forward with this for now?

Reply:
I created the [.github repo](https://github.com/lifeomic/.github) and followed the tutorial but am not seeing the workflow as an option when I go to any repo in the org and set up a new workflow via the Actions tab. 

I'm not sure what the deal is but I haven't been able to Google up any troubleshooting since it's a new feature.

Reply:
@mschroering @hemp I did some more digging and found out that our `.github` repo [must be public in order to use the workflows](https://github.community/t/github-repo-workflow-templates-only-public/126583/3) defined there across the organization. Is this something we want to do? The repo will only contain workflow templates and I don't envision there being anything sensitive inside them, but it's hard to say at this point.

Reply:
Seems ok given this is a public repo too and the information will end up here.

Reply:
I did the above and am closing this in favor of PR #89.